src/sessions.ts

```diff
@@ -703,11 +704,11 @@ function endTransaction(
   }
 
   if (txnState === TxnState.TRANSACTION_COMMITTED) {
-    writeConcern = Object.assign({ wtimeout: 10000 }, writeConcern, { w: 'majority' });
+    writeConcern = Object.assign({ wtimeoutMS: 10000 }, writeConcern, { w: 'majority' });
```

Comment:
Should we call `WriteConcern.fromOptions` here? and above as well, These Object.assign calls would make something that's an instance of the WriteConcern class turn into a plain object.

Reply:
It's expected in the sessions module that the `writeConcern` is not an instance of a `WriteConcern` object since it's going to be put directly into the options. That worked previously is it was an instance since the `WriteConcern` properties matched 1-1 with what was valid to pass to the server, but this is no longer the case. `wtimeoutMS` and `journal` are not valid (they must be `wtimeout` and `j` when passed to the server). The reason for the change on this line is that later when we call `WriteConcern.apply()`, which always happens if a write concern exists, it's looking for our non-deprecated client facing option names.

Reply:
I think it's cleaner here to do it this way here as `writeConcern` in this context (from line 699) is not an instance of a `WriteConcern` object but rather a plain object.